File: GEOMETRIA/1292/main.dart
import 'dart:io';
import 'dart:math';

double largestSide(double f) {
  return f * sin(108 * pi / 180) / sin(63 * pi / 180);
}

void main() {
  String? line;
  while ((line = stdin.readLineSync()) != null) {
    double f = double.parse(line!);
    print(largestSide(f).toStringAsFixed(10));
  }
}

File: GEOMETRIA/1292/main.js
function largestSide(f) {
    return f * Math.sin(108 * Math.PI / 180) / Math.sin(63 * Math.PI / 180);
}

let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').trim().split('\n');
for (let line of lines) {
    let f = parseFloat(line);
    console.log(`${largestSide(f).toFixed(10)}`);
}
